src/directives/mlhrTable.js

```diff
@@ -244,9 +245,12 @@ angular.module('datatorrent.mlhrTable.directives.mlhrTable', [
     if (angular.isObject(scope.options.loadingPromise) && typeof scope.options.loadingPromise.then === 'function') {
       scope.options.loadingPromise.then(
         function(){
+          scope.options.loadingError = false;
           scope.api.setLoading(false);
         },
         function(reason){
+          scope.options.loadingError = true;
+          scope.options.loadingErrorMessage = 'Failed loading table data: ' + reason;
```

Comment:
Rather than hardcode `loadingErrorMessage` here, make it an option like `loadingText` is (call it `errorText`).